config/ui-extensions/eventing/general

```diff
@@ -0,0 +1,15 @@
+resource:
+  kind: Eventing
+  group: operator.kyma-project.io
```

Comment:
please change the name

Reply:
@k15r [Telemetry](https://github.com/kyma-project/telemetry-manager/blob/3ed3e38fc12408347388d3355e4ee338fa836485/config/busola/telemetry_busola_extension_cm.yaml#L103C38-L103C38) and [KEDA](https://github.com/kyma-project/keda-manager/blob/0f0989db147114a68f9db2fba534977451d1946e/config/ui-extensions/kedas/general#L3C5-L3C5) modules seem to have the same with ours. 
What value did you want and why do you want it?

Reply:
Sorry, wrong line :)